### ISSUE TITLE
[BottomNav][Android] Changing tabs messes up text color of tabs

### ISSUE THREAD

**Issue by PeterStaev** [NativeScript/NativeScript#7623]
**Environment**
Provide version numbers for the following components (information can be retrieved by running `tns info` in your project folder or by inspecting the `package.json` of the project):
 - CLI: 6.0.0
 - Cross-platform modules: 6.0.2
 - Android Runtime: 6.0.0
 - iOS Runtime: 6.0.0
 - Plugin(s):

**Describe the bug**
This is a really strange issue. When the BottomNav is loaded all looks correctly, But when you change the tab selection, the font of the text  in the tabs changes to some strange color that seems to have some alpha applied. 

**To Reproduce**
https://play.nativescript.org/?template=play-tsc&id=Da7vym&v=3
* On load you will notice that all tabs have correct colors as per css - the selected one has gold, while the others have red. 
* Tap on the 1st tab. This messes up the 3rd tab color while the other two have correct colors
* Tap on the 3rd tab. This messes up the color of the 2nd tab.

To sum up - the `:active:` color is correctly applied but the "normal" color is applied only to the previously selected tab. All other tabs have a messed up color with some strange transparency. 

**Expected behavior**
Styles to be applied correctly to all tabs. 

Related to #6967.
